test/e2e/framework/pod/pod_client.go

```diff
@@ -89,19 +95,41 @@ func PodClientNS(f *framework.Framework, namespace string) *PodClient {
 type PodClient struct {
 	f *framework.Framework
 	v1core.PodInterface
-	namespace string
+	namespace  string
+	trackOwner bool
+}
+
+// WithOwnerTracking enable automatic add of annotations recording the code location
+// which created a pod. This is helpful when troubleshooting e2e tests (like e2e_node)
+// which leak pods because insufficient cleanup.
+// Note we want a shallow clone to avoid mutating the receiver.
+// Only PodClients returned by this helper should have tracking enabled.
+func (c PodClient) WithOwnerTracking() *PodClient {
```

Comment:
I prefer a boolean parameter instead of adding WithoutOwnerTracking. That's what is done elsewhere.

Reply:
Also cross-reference the global default here with `[GlobalTrackOwner]`.

Reply:
Sorry, missed the convention. Fixing.